### PR TITLE
initial recipe of SPINGO version 1.3

### PR DESCRIPTION
**Pull request by Takadonet**
* [X] I have read the guidelines above.
* [X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/spingo/build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+
+
+
+set -x -e
+
+export CC=${PREFIX}/bin/gcc
```
Comment (bgruening) [inline recipes/spingo/build.sh]: This should not be needed.

File: recipes/spingo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+about:
+  home: https://github.com/GuyAllard/SPINGO
+  license: ''
```
Comment (bgruening) [inline recipes/spingo/meta.yaml]: License is https://github.com/GuyAllard/SPINGO/blob/master/LICENSE

File: recipes/spingo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  summary: 'SPecies level IdentificatioN of metaGenOmic amplicons'
+build:
+  number: 0
+  string: py{{CONDA_PY}}_boost{{CONDA_BOOST}}
+package:
+  name: spingo
+  version: 'v1.3'
```
Comment (bgruening) [inline recipes/spingo/meta.yaml]: can we just call it 1.3

File: recipes/spingo/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+requirements:
+  build:
+      - gcc
+      - libgcc >=3.4.15
```
Comment (bgruening) [inline recipes/spingo/meta.yaml]: can you follow our guideline here: https://bioconda.github.io/guidelines.html#c-c

File: recipes/spingo/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+      - boost {{CONDA_BOOST}}*
+  run:
+      - libgcc >=3.4.15
```
Comment (bgruening) [inline recipes/spingo/meta.yaml]: no need to specify the version here

File: recipes/spingo/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+      - boost {{CONDA_BOOST}}*
+source:
+  git_rev: v1.3
+  git_url: https://github.com/GuyAllard/SPINGO.git
```
Comment (bgruening) [inline recipes/spingo/meta.yaml]: Can you please user here a tarball and a checksum. With this we can backup the tarballs better.

File: recipes/spingo/build.sh (hunk 1 above)
Comment (Takadonet) [inline recipes/spingo/build.sh]: Removed.

File: recipes/spingo/meta.yaml (hunk 2 above)
Comment (Takadonet) [inline recipes/spingo/meta.yaml]: Added.

File: recipes/spingo/meta.yaml (hunk 3 above)
Comment (Takadonet) [inline recipes/spingo/meta.yaml]: Sounds good.

File: recipes/spingo/meta.yaml (hunk 4 above)
Comment (Takadonet) [inline recipes/spingo/meta.yaml]: Changed to follow guidelines.

File: recipes/spingo/meta.yaml (hunk 5 above)
Comment (Takadonet) [inline recipes/spingo/meta.yaml]: removed.

File: recipes/spingo/meta.yaml (hunk 6 above)
Comment (Takadonet) [inline recipes/spingo/meta.yaml]: Changed to tar.gz and added checksum.

**Comment by Takadonet** [comment]
Waiting on Travis CI testing.  Thanks for all the quick feedback @bgruening

**Comment by bgruening** [comment]
Sure thing!

**Pull request merged**